kw-completion.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+complete -W "export build install prepare new ssh mail mount
+	     umount boot vars up codestyle maintainers help" kw
```

Comment:
I recommend you to put this line in the file "kw.sh" on top of "function kw()", with this change you can get rid of the file "kw-completion.sh". Additionally, I recommend you add the short commands in the list as well.

Reply:
Okay! Regarding the complete command, I found that it is a builtin function. Does it need to add in the dependence?

Reply:
https://www.gnu.org/software/bash/manual/html_node/Programmable-Completion-Builtins.html